<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownOptDialog</class>
 <widget class="QWidget" name="DownOptDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="panel1" native="true">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text" stdset="0">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0">
       <widget class="QLabel" name="label5">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Date Format for Test</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QPushButton" name="btnUrlFile">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="cBKeepErr">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string> Keep Error Info on Download Error (*.err)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="cBKeepList">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string> Keep Remote Directory Listings (.listing)</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="btnLogFile">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QLabel" name="label1">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>URL List File for GNSS Data</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label2">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Output Debug Trace</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label6">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string># Columns for Test</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cBDateFormat">
        <property name="toolTip">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>YYYY-DOY</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>YYYY/mm/dd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Week</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="10" column="0" colspan="3">
       <widget class="QLineEdit" name="lEUrlFilename">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label3">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Proxy Address</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label4">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Download Log File</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="cBLogAppend">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Append</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="3">
       <widget class="QLineEdit" name="lELogFilename">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="cBTraceLevel">
        <property name="toolTip">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>OFF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Level 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Level 2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Level 3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Level 4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Level 5</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="QLineEdit" name="lEProxy">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="sBTestColumnCount">
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnOk">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;OK</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cBKeepErr</tabstop>
  <tabstop>cBKeepList</tabstop>
  <tabstop>sBTestColumnCount</tabstop>
  <tabstop>cBDateFormat</tabstop>
  <tabstop>cBTraceLevel</tabstop>
  <tabstop>lEProxy</tabstop>
  <tabstop>cBLogAppend</tabstop>
  <tabstop>btnLogFile</tabstop>
  <tabstop>lELogFilename</tabstop>
  <tabstop>btnUrlFile</tabstop>
  <tabstop>lEUrlFilename</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
